Canvas/TechDebt/Files and Folders/FileViewController.m

```diff
@@ -302,7 +302,7 @@ - (UIView *)legacyFileMessageView {
 
 - (void)addOpenInARButton {
     UIButton *button = [UIButton new];
-    [button setTitle:NSLocalizedStringFromTableInBundle(@"Open in AR", nil, [NSBundle bundleForClass: self.class], nil) forState:UIControlStateNormal];
+    [button setTitle:NSLocalizedStringFromTableInBundle(@"Augment Reality", nil, [NSBundle bundleForClass: self.class], nil) forState:UIControlStateNormal];
```

Comment:
😆 Is this really what they want the title to be?

Reply:
Haha yeah